src/dfcx_scrapi/core/entity_types.py

```diff
@@ -187,14 +212,14 @@ def entity_types_to_df(
 
 
     def get_entities_map(self, agent_id: str = None, reverse=False):
-        """Exports Agent Entityt Names and UUIDs into a user friendly dict.
+        """Exports Agent Entity Names and UUIDs into a user friendly dict.
 
         Args:
-          - agent_id, the formatted CX Agent ID to use
-          - reverse, (Optional) Boolean flag to swap key:value -> value:key
+          agent_id: the formatted CX Agent ID to use
+          reverse: (Optional) Boolean flag to swap key:value -> value:key
 
         Returns:
-          - intents_map, Dictionary containing Entity UUIDs as keys and
+          Dictionary containing Entity UUIDs as keys and
               intent.display_name as values
```

Comment:
Good catch, I missed that one

Reply:
Also noticed this one said Entity rather than Entity Type, which may have been a bit confusing, since it contains the entity type names and UUIDs, and not the entities within them.

Reply:
@SeanScripts Ack. `Entity Type` is the correct semantics to use here.